### ISSUE TITLE
Syntax error in feed despite validation

### ISSUE THREAD

**Issue by madduck** [wking/rss2email#71]
See http://feedvalidator.org/check.cgi?url=http://www.flassbeck-economics.de/feed/ to confirm that the site serves valid RSS/XML. However, when processing, rss2email prints:

```
sax parsing error: <unknown>:6:0: syntax error: flassbeck-economics.de (http://www.flassbeck-economics.de/feed/ -> flassbeck-economics.de@example.com)
```

and fails.

**Comment by jwilk** [comment]
The server doesn't seem to like rss2email's User-Agent:

```
>>> import feedparser
>>> d = feedparser.parse('http://www.flassbeck-economics.de/feed/')
>>> d.headers['Content-Type']
'application/rss+xml; charset=UTF-8'
>>> feedparser.USER_AGENT = 'rss2email/3.9 (https://github.com/wking/rss2email)'
>>> d = feedparser.parse('http://www.flassbeck-economics.de/feed/')
>>> d.headers['Content-Type']
'text/html; charset=utf-8'
>>> str(d.bozo_exception)
'<unknown>:7:0: syntax error'
```

**Comment by madduck** [comment]
Oh, nice catch. I'll let the operators know.

Meanwhile, maybe it'd be worth considering allowing the user-agent to be overridden? Would this be worthwhile?

**Comment by jwilk** [comment]
NB, there should have been a warning about server sending you HTML. I've filed #83 about this.
There's PR #76 to make User-Agent customizable.

**Comment by Ekleog** [comment]
Hello,

This repository has been deprecated for a few years now, and has been replaced by https://github.com/rss2email/rss2email .

If this issue is still relevant to you, and not fixed with v3.12.2, could you please reopen the issue there?

Cheers,
  Leo

**Comment by madduck** [comment]
This issue does not seem to happen with the new version anymore.
